crates/rooch-executor/src/actor/executor.rs

```diff
@@ -55,34 +52,22 @@ type ValidateAuthenticatorResult = Result<
 impl ExecutorActor {
     pub fn new(
         root: RootObjectEntity,
-        mut genesis: RoochGenesis,
         moveos_store: MoveOSStore,
         rooch_store: RoochStore,
     ) -> Result<Self> {
-        //TODO refactor the gas schedule loading
         let resolver = RootObjectResolver::new(root.clone(), &moveos_store);
-        let gas_schedule_entries = get_gas_schedule_entries(&resolver);
-        if let Some(gas_entries) = gas_schedule_entries {
-            if let Some(gas_parameters) =
-                rooch_framework::natives::NativeGasParameters::from_on_chain_gas_schedule(
-                    &gas_entries,
-                )
-            {
-                genesis.rooch_framework_gas_params = gas_parameters;
-            }
-        }
+        let gas_parameters = FrameworksGasParameters::load_from_chain(&resolver)?;
```

Comment:
Maybe I have missed something, I cannot find the code for the function FrameworksGasParameters::load_from_chain().

Reply:
https://github.com/rooch-network/rooch/blob/71705dc2c82d543aa2059e0b2dbc2cd570c07fbe/crates/rooch-genesis/src/lib.rs#L89-L96